lib/sys-info.ts

```diff
@@ -340,6 +344,57 @@ export class SysInfo implements NativeScriptDoctor.ISysInfo {
 		this.shouldCache = shouldCache;
 	}
 
+	public getGitPath(): Promise<string> {
+		return this.hostInfo.isWindows ? this.findGitWin32() : this.findGitUnix();
+	}
+
+	private async findGitWin32(): Promise<string> {
+		let result: string;
+		const win32Paths = [process.env["ProgramFiles"], process.env["ProgramFiles(x86)"]];
+		for (const win32Path of win32Paths) {
+			result = this.findSystemGitWin32(win32Path);
```

Comment:
on x86 environment you will not have `process.env("ProgramFiles(x86)")` variable and the call to `findSystemGitWin32` will fail with `path must be a string, received undefined`. Consider handling it here or in the method itself